styles.css

```diff
@@ -0,0 +1,126 @@
+/* Styl pro tabulku */
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+
+/* Styl pro hlavičku a patičku tabulky */
+thead, tfoot {
+    font-weight: bold;
+    border: 2px solid black;
+    background-color: #e0e0e0; /* Light gray background for better visibility */
+}
+
+/* Přidání rámečků pro hlavičku tabulky */
+thead tr, tfoot tr {
+    border: 2px solid black;
+}
+
+/* Styl pro jednotlivé buňky v hlavičce */
+th {
+    padding: 10px;
+    text-align: left;
+    border-bottom: 2px solid black;
+}
+
+/* Styl pro patičku tabulky */
+tfoot td {
+    border-top: 2px solid black;
+    text-align: center;
+    padding: 8px;
+}
+
+/* Styl pro sudé a liché řádky tabulky */
+tbody tr:nth-child(even) {
+    background-color: #f9f9f9;
+}
+
+tbody tr:nth-child(odd) {
+    background-color: #ffffff;
+}
+
+/* Hover efekt pro řádky tabulky */
+tbody tr:hover {
+    background-color: #dcdcdc;
+}
+
+/* Skrytí rámečků pro jednotlivé buňky v těle tabulky */
+tbody td {
+    border: none;
+    padding: 8px;
+}
+
+/* === FORMULÁŘ === */
+
+/* Obecné styly pro formulář */
+form {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+    padding: 15px;
+    max-width: 600px;
```

Comment:
ten `max-width` ti zbytecne omezuje sirku formulare a pak to nefunguje idealne, protoze i na velkem displeji je to relativne male (byt vetsinou se omezuje sirka a nedovoluje napr. 100% protoze na velkem displeji to vypada pak spatne, tak 600px je za mne pak naopak zase malo :)